tests/phpunit/Admin/PluginActionLinksTest.php

```
@@ -0,0 +1,200 @@
+		// Mock the edac_link_wrapper function if it doesn't exist.
+		if ( ! function_exists( 'edac_link_wrapper' ) ) {
+			/**
+			 * Mock the edac_link_wrapper function.
+			 *
+			 * @param string $url      The URL to wrap.
+			 * @param string $source   The source parameter.
+			 * @param string $campaign The campaign parameter.
+			 * @param bool   $unused   Unused parameter for compatibility.
+			 * @return string The wrapped URL.
+			 */
+			function edac_link_wrapper( $url, $source, $campaign, $unused ) { // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable
```

PHPMD (2.15.0)

Avoid unused parameters such as '$unused'. (Unused Code Rules)

(UnusedFormalParameter)